go.mod

```diff
@@ -5,12 +5,7 @@ go 1.17
 require (
 	github.com/codahale/rfc6979 v0.0.0-20141003034818-6a90f24967eb
 	github.com/shibumi/go-pathspec v1.2.0
+	github.com/spf13/cobra v1.2.1
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/sys v0.0.0-20210616094352-59db8d763f22
-)
-
-require (
-	github.com/davecgh/go-spew v1.1.0 // indirect
-	github.com/pmezard/go-difflib v1.0.0 // indirect
-	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
```

Comment:
I wonder if this is tripping up the builds. 1.17 added some changes around tracking indirect references that shows up as a separate require section. Try reverting this file, bumping up your local go install to 1.17, and then running `go mod tidy -go=1.17`. For me the file looks like (from an earlier commit):

```
module github.com/in-toto/in-toto-golang

go 1.17

require (
        github.com/codahale/rfc6979 v0.0.0-20141003034818-6a90f24967eb
        github.com/shibumi/go-pathspec v1.2.0
        github.com/spf13/cobra v1.2.1
        github.com/stretchr/testify v1.7.0
        golang.org/x/sys v0.0.0-20210909193231-528a39cd75f3
)

require (
        github.com/davecgh/go-spew v1.1.1 // indirect
        github.com/inconshreveable/mousetrap v1.0.0 // indirect
        github.com/pmezard/go-difflib v1.0.0 // indirect
        github.com/spf13/pflag v1.0.5 // indirect
        gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
)
```

Reply:
Reference to the 1.17 change: https://golang.org/doc/go1.17#go-command

Reply:
That worked! Thank you for looking into it. Apparently the path to my local go was pointing to an old install of go on version 1.16. Using 1.17 resolved the issue.